mParticle-BranchMetrics/MPKitBranchMetrics.m

```diff
@@ -18,186 +18,579 @@
 
 #import "MPKitBranchMetrics.h"
 #import <Branch/Branch.h>
-
 #if TARGET_OS_IOS == 1 && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0
-    #import <UserNotifications/UserNotifications.h>
-    #import <UserNotifications/UNUserNotificationCenter.h>
+#import <UserNotifications/UserNotifications.h>
+#import <UserNotifications/UNUserNotificationCenter.h>
 #endif
 
+__attribute__((constructor))
+void MPKitBranchMetricsLoadClass(void) {
```

Comment:
In some project configurations, the runtime linker won't figure out that `MPKitBranchMetrics` is a dependency, so it won't load `MPKitBranchMetrics`, and the `+ load` method won't be called.

This is a sort of dumb work around to make sure the class is loaded. Since this function lives in the same file as the class implementation, the linker will load everything in the file.